tests/wallet/nft_wallet/test_nft_1_offers.py

```diff
@@ -1316,12 +1326,23 @@ async def test_complex_nft_offer(
     for i in range(0, 2):
         await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph_maker))
         await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph_taker))
-    await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph_taker))
+    if not forwards_compat:
+        for i in range(9):
```

Comment:
Why do we need 9 blocks now?  The amounts still seem pretty small yeah?

Reply:
because >100% royalty gets expensive